.travis.yml

```diff
@@ -125,13 +125,12 @@ install:
 script:
   # set git user info     
   - git config --global user.email "blt-dev@llnl.gov" 
-  - git config --global user.name  "BLT Robot"    
+  - git config --global user.name  "BLT Robot"
+  - BLT_SOURCE_DIR=${TRAVIS_BUILD_DIR}
   - cmake --version
   - cd $TRAVIS_BUILD_DIR/..
   # copy our internal testing project
   - cp -r blt/cmake/blt-test .
-  # symlink blt into this project
-  - ln -s ${TRAVIS_BUILD_DIR} blt-test/blt
```

Comment:
Thanks -- the symbolic link with recursive directories was awkward.

Reply:
no problem, glad we can stop & avoid fun with sphinx infinity traversing the recursive symlink.